src/test/scala/TextToStructuredDataSpec.scala

```diff
@@ -0,0 +1,140 @@
+import com.github.mrpowers.spark.fast.tests.DatasetComparer
+import com.typesafe.scalalogging.LazyLogging
+import org.scalatest.FunSpec
+import TextToStructuredData.getStructuredDataFromText
+import org.apache.spark.sql.Row
+
+//TODO: switch to more concise and powerful spark-testing-base
+class TextToStructuredDataSpec extends FunSpec with SparkSessionCreator with LazyLogging with DatasetComparer {
+  import spark.implicits._
+
+  it("Simple general check") {
```

Comment:
Please apply use-case/corner-case descriptive naming to tests. Take a look [here](http://www.scalatest.org/user_guide/selecting_a_style) for a examples.
Ideally, one should be able to reason what kind of features an implementation supports by looking at test cases. So, if the test cases represent use-cases, it becomes clear.

Reply:
I tried to fix it.